content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -626,6 +627,10 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `NetworkPolicyEndPort`: Enable use of the field `endPort` in NetworkPolicy objects,
   allowing the selection of a port range instead of a single port.
 - `NetworkPolicyStatus`: Enable the `status` subresource for NetworkPolicy objects.
+- `NewVolumeManagerReconstruction`: Enables improved discovery of mounted volumes during kubelet
+  startup. Since this code has been significantly refactored, we allow to opt-out in case kubelet
+  gets stuck at the startup or is not unmounting volumes from terminated Pods. Note that this
+  refactoring was behind `SELinuxMountReadWriteOncePod` alpha feature gate in Kubernets 1.25.
```

Comment:
* We should revise the `SELinuxMountReadWriteOncePod` description too, then.
  * We can backport a change to 1.25 and 1.26 docs to explain that the feature gate behavior partly moves to `NewVolumeManagerReconstruction`
* “Kubernets” is a typo